flake.nix

```diff
@@ -777,6 +783,13 @@
                 + wasm32CrossEnvVars;
             });
 
+            # Like `cross` but only with wasm
```

Comment:
`link`?

Reply:
The shell right above this one is called `cross`.